# Goal
geometry_msgs/Point target_position
float64 duration

---
# Result
bool success
geometry_msgs/Point final_position

---
# Feedback
geometry_msgs/Point position_error
